server/test/test_main.py

```diff
@@ -15,7 +15,44 @@ def test_make_session():
     assert response.status_code == 201
 
 
-def test_websocket_connection():
+def test_make_session_invalid_authors():
```

Comment:
nice tests, I recommend following the folder structure in the source code.

Like the web socket stuff is in gitgame/routes/ws.py

You could put this in test/route/ws.py